src/Options.js

```diff
@@ -59,7 +61,9 @@ class Options {
     size: number;
     textSize: number;
     phantom: boolean;
-    font: string | void;
+    fontFamily: string | void;
```

Comment:
I think it's fine leaving this as is, but we might want to change this to `fontFamily: string` in a follow up diff to match `fontWeight` and `fontShape`.  Could you add a `TODO`?

Reply:
Done